flow/embedded_views.h

```diff
@@ -201,6 +201,33 @@ class EmbeddedViewParams {
            sizePoints == other.sizePoints &&
            mutatorsStack == other.mutatorsStack;
   }
+
+  // Clippings are ignored in this operation
+  // TODO(cyanglaz): Factoring clipping.
+  // https://github.com/flutter/flutter/issues/59821
+  SkRect GetBoundingRectAfterMutations() const {
+    SkPath path;
+    SkRect starting_rect = SkRect::MakeSize(sizePoints);
```

Comment:
does it still need the `offsetPixels`?

Reply:
I think mutator stack should contain all the offset informations already, By querying through the stack, we have the final offset